### DISCUSSION TITLE
Can't locate external hard drive inside UI

### DISCUSSION THREAD

**Discussion by And-chan** [qarmin/czkawka#1166] [Q&A]
Hi, I just got the program running and I need to scan a folder on my external hard drive but I can't find it on the add directory menu, any suggestions?

EDIT: I'm on PopOS

**Comment by qarmin** [comment]
App uses default system(or gtk) version of file dialog, so if it is not visible there, then probably this drives are not visible also in other apps that uses same mechanism.

You can check via e.g. gnome-disks path to mounted disks(at screenshoot is /media/rafal/KINGSTON) and manually check if0 path are available/visible

![Screenshot from 2023-12-25 22-39-08](https://github.com/qarmin/czkawka/assets/41945903/2ce42a30-280b-4e64-a717-204e6ac7c768)

Snap version require additional commands to access external storage

**Reply by FoxyLevin** [reply]
Hey @qarmin, 
thx for czkawka! I have the same issue like @And-chan on Ubuntu 24.04 Mate [edit: I'm using the snap-version and the device is visible to other apps]. What exactly are the additional commands to access external storage?

**Comment by hendrickhalim** [comment]
I also had the same problem. I used the snap package. Decided to download the app image package instead. It works well. I would suggest the app image package. The snap package did not work.

**Comment by Behv730** [comment]
I used the snap package and for it worked to put `sudo snap connect czkawka:removable-media` and then putting the paths in manually. This I got from Google Gemini.
